compiler/luci-interpreter/src/loader/CMakeLists.txt

```diff
@@ -10,6 +10,13 @@ set(SOURCES
     RuntimeToIR.h
     nodes/Builders.h)
 
+# include kernel specific builders
+macro(REGISTER_KERNEL NODE)
+  list(APPEND SOURCES "nodes/${NODE}.cpp")
+endmacro(REGISTER_KERNEL)
+include(${KERNEL_REGISTER_FILE})
+message("SOURCES" ${SOURCES})
```

Comment:
@binarman Do we need this msg here or better to make clean up PR? =)

Reply:
Personally, I would like to remove this line.

Reply:
Personally, I would like to remove this line =)

Reply:
this is a debug leftover, I noticed it too after merge =)
It should be removed, merge was not intentional